pandas_gbq/tests/test_gbq.py

```diff
@@ -16,19 +17,15 @@
 from pandas import NaT
 from pandas.compat import u, range
 from pandas.core.frame import DataFrame
-import pandas.io.gbq as gbq
+from pandas_gbq import gbq
 import pandas.util.testing as tm
 from pandas.compat.numpy import np_datetime64_compat
 
 PROJECT_ID = None
 PRIVATE_KEY_JSON_PATH = None
 PRIVATE_KEY_JSON_CONTENTS = None
 
-if compat.PY3:
-    DATASET_ID = 'pydata_pandas_bq_testing_py3'
-else:
-    DATASET_ID = 'pydata_pandas_bq_testing_py2'
-
+DATASET_ID = 'pydata_pandas_gbq_py' + ''.join(map(str, sys.version_info))
```

Comment:
sure, though we *may* want to have an env variable from the travis file (or default it to this if not on travis)